dip-0008.md

```diff
@@ -33,6 +33,12 @@ of blocks and finding near-instant consensus on the longest valid/accepted
 chain. ChainLocks leverages LLMQ Signing Requests/Sessions to accomplish
 this.
 
+[The original proposal by Satoshi Nakamoto](https://www.bitcoin.com/bitcoin.pdf) 
+provided a novel solution to the Byzantine general's problem.  In some publications, 
+this new consensuses mechanism is called a Nakamoto Consensus.  ChainLocks provide for 
```

Comment:
```suggestion
this new consensuses mechanism is called a Nakamoto Consensus.  ChainLocks provides for 
```
Although "ChainLocks" ends with an 's', it's normally been treated as a singular noun